servicepulse/install-servicepulse-in-iis.md

```diff
@@ -57,45 +55,9 @@ netsh http delete urlacl http://+:9090/
 > [!NOTE]
 > If TLS is to be applied to ServicePulse then ServiceControl also must be configured for TLS. This can be achieved by reverse proxying ServiceControl through IIS as outlined below.
```

Comment:
can "below" have a link to the document section?

Reply:
```suggestion
> If TLS is to be applied to ServicePulse then ServiceControl also must be configured for TLS. This can be achieved by reverse proxying ServiceControl through IIS as outlined [below](#servicecontrol).
```